srv/salt/_modules/dg.py

```diff
@@ -597,6 +603,20 @@ def limit(self) -> int:
         """
         return self.data_device_attrs.get("limit", 0)
 
+    @property
+    def format(self) -> str:
+        """
+        On-disk-format - Filestore/Bluestore
+        """
+        return self.filter_args.get("format", "bluestore")
+
+    @property
+    def journal_size(self) -> int:
+        """
+        Journal size
+        """
+        return self.filter_args.get("journal_size", False)
```

Comment:
Should the default here be `0`, rather than `False`, to match the specified return type of the function (int)?